### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by Ananta-Nath-0312** [wahyu9kdl/wahyu9kdl.github.io#5] [Q&A]
Hi is the project still open to contribute? If yes, what are the system requirements for me to run it locally?

**Comment by wahyu9kdl** [comment]
# 
Yes Ofcourse you can contribute to our projects.
Requirements are quite simple:

- My Profile : https://wahyu9kdl.github.io

1. You can clone the Git repo: 

- HTTPS 

```
https://github.com/wahyu9kdl/wahyu9kdl.github.io.git 
```
- SSH 

```
git@github.com:wahyu9kdl/wahyu9kdl.github.io.git
```
Github CLI 

```
gh repo clone wahyu9kdl/wahyu9kdl.github.io 
```

- Website : 
https://www.awdev.eu.org https://www.alhikmah.my.id
- Projects
wahyu9kdl.github.io
License
 MIT license
 
2. Download the file and you can edit and publish it.
3. Please Support Our Projects by donating via:
- Github Sponsor
https://github.com/sponsors/wahyu9kdl

💝 [OPEN SPONSORED](https://github.com/sponsors/wahyu9kdl)

- patreon
https://patreon.com/Awfanspage
-open_collective
https://opencollective.com/wahyu9kdl
or 
- https://paypal.me/wahyudi9kdl
- https://trakteer.id/awfanspage/tip
- https://saweria.co/AwGroupChannel
- https://www.patreon.com/join/Awfanspage

4. ♥️ Thank you 😊🎨
